src/components/BlockSection/BlockSection.tsx

```diff
@@ -6,33 +6,31 @@ import { ListItem } from '@components/List/Item'
 import { Panel } from '@components/Panel'
 import { Paragraph } from '@components/Paragraph'
 import { Spacer } from '@components/Spacer'
+import { generateFakeUuid } from '@helpers/data'
 import { useStyles } from '@helpers/hooks/useStyles'
-import PropTypes from 'prop-types'
-import uuid from 'react-uuid'
+import { FC, MouseEvent } from 'react'
 
 import styles from './BlockSection.module.scss'
 
-/**
- * Block Section - stateless presentational component
- * @param {object} props - props
- * @param {string|array} props.className - list of class names
- * @param {string} props.title - title
- * @param {string} props.message - message
- * @param {string} props.button - button title
- * @param {func} props.onClick - button action
- * @param {string} props.url - image url
- * @param {bool} props.reverse - column reverse
- * @param {array} props.list - list
- * @return {object} An object of children element
- */
-const BlockSection = ({
-  className,
+interface BlockSectionProps {
+  className?: string | string[]
+  title: string | string[]
+  message: string | string[]
```

Comment:
title i message, zrobiłbym sam type string
Wiem, że w PropTypes było też array ale raczej nie używamy tego w ten sposób to dobry moment żeby takie rzeczy poprawiać :P

Reply:
Okej. Leci następny PR 👍🏻